app/Http/Controllers/Api/RoleController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Api\RolesRequest;
+use Spatie\Permission\Models\Permission;
+use Spatie\Permission\Models\Role;
+
+class RoleController extends Controller
+{
+    public function index()
+    {
+        $roles = Role::all();
+        $permissions = Permission::all();
+
+        return response()->json([
+            'roles' => $roles,
+            'permissions' => $permissions,
```

Comment:
why are we sending permissions in role list API?

Reply:
Because when we will assign permissions to roles from frontend then we also need to show list of permissions in frontend so I am passing permissions along with roles. Please let me know, Do I need to create a seperate API for the permission list?

Reply:
Yes, let's create a separate API for permissions, if we want the list of permissions.

Here, it will make sense only if you're returning permissions for every role.